core/lib/Cardano/Script.hs

```diff
@@ -212,3 +220,68 @@ blake2b224 =
 -- Size, in bytes, of a hash of public key (without the corresponding chain code)
 hashSize :: Int
 hashSize = hashDigestSize Blake2b_224
+
+-- Examples of Script jsons:
+--{ "key" : "e09d36c79dec9bd1b3d9e152247701cd0bb860b5ebfd1de8abb6735a" }
```

Comment:
Rather than a json object, could this just be a string?
The label "key" is not quite right because it's a hex-encoded hash of a key.
And were we going to use bech32 for key hashes?

Reply:
good idea, I was also thinking about it (ie. to express verification key hash as string). also in accordance to "line parser" added all, bech32, base16 and base58, support. see https://github.com/input-output-hk/cardano-addresses/pull/84/commits/3c2bb9a1e8743c88bddb7ab271bce8fe393eea09